README.md

```diff
@@ -188,6 +188,7 @@ contact the [JWST Help Desk](https://jwsthelp.stsci.edu).
 
 | jwst tag | DMS build | CRDS_CONTEXT |   Date     |          Notes                                |
 | -------- | --------- | ------------ | ---------- | ----------------------------------------------|
+|  1.4.3   | B7.9      | 0800         | 2022-02-03 | Final B7.9                                    |
```

Comment:
This should be labeled as B7.9.1 in the Notes column.

Reply:
Done and ready to merge. Tests passed on previous commit